geopaparazzilibrary/src/main/java/eu/geopaparazzi/library/network/upload/MultipartUtility.java

```diff
@@ -0,0 +1,223 @@
+package eu.geopaparazzi.library.network.upload;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLConnection;
+
+import eu.geopaparazzi.library.database.GPLog;
+import eu.geopaparazzi.library.network.NetworkUtilities;
+import eu.geopaparazzi.library.network.upload.UploadFileIntentService;
+
+/**
+ * This utility class provides an abstraction layer for sending multipart HTTP
+ * POST requests to a web server.
+ * @author www.codejava.net
+ * https://gist.github.com/Antarix/a36faeaff3092b1fd977
+ */
```

Comment:
Hi @brentfraser , is there a way to understand teh license of this? If I am correct in case there is no license defined, you are not allowed to copy from a gist by default.